tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -154,6 +154,10 @@ protected override bool Skip (Type type, string protocolName)
 				case "AVMutableComposition":
 				case "AVCompositionTrackSegment":
 				case "MKMapSnapshotOptions":
+				case "NSLayoutAnchor`1":
+				case "NSLayoutDimension":
+				case "NSLayoutXAxisAnchor":
+				case "NSLayoutYAxisAnchor":
```

Comment:
if those types exists in **macOS** then the `case`s must be moved into the shared `ApiProtocolTest.cs` and not the iOS specific file. That might be why you have issues with the mac tests...